dev-docs/bidders/sharethrough.md

```diff
@@ -5,7 +5,9 @@ biddercode: sharethrough
 description: Prebid Sharethrough Adaptor
 gdpr_supported: true
 coppa_supported: true
-media_types: native
+floors_supported: true
+media_types: video
```

Comment:
```suggestion
media_types: banner, video
```

According to your prebid adapter
https://github.com/prebid/Prebid.js/blob/2740872e8def571ab4f1142fe1235ac5bf8a439b/modules/sharethroughBidAdapter.js#L4

you also support banner

Reply:
did look into this, looks like the docs show 'display' by default so we should be covered there